package.json

```diff
@@ -9,14 +9,16 @@
     "@testing-library/jest-dom": "^5.16.5",
     "@testing-library/react": "^13.4.0",
     "@testing-library/user-event": "^13.5.0",
+    "firebase": "^9.21.0",
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
+    "react-native": "^0.71.8",
     "react-router-dom": "^6.11.1",
     "react-scripts": "5.0.1",
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "HTTPS=true SSL_CRT_FILE=./ssl-certificates/localhost+2.pem SSL_KEY_FILE=./ssl-certificates/localhost+2-key.pem react-scripts start",
+    "start": "set HTTPS=true && set SSL_CRT_FILE=./ssl-certificates/localhost.pem && set SSL_KEY_FILE=./ssl-certificates/localhost-key.pem && react-scripts start",
```

Comment:
Did you change this for your Windows environment? You can change it locally; however, you should change it back before using git to commit it.

Reply:
Yes, I had to change for Windows environment. I'll change it before pushing again

Reply:
You changed this configuration again. Please do some review before pushing code.

Reply:
Oh, whoops. I kept having to change it for me to test. Will fix and push again.